### PR TITLE
Swap Ricochet and Wire

### PR DESCRIPTION
**Pull request by Perelandra0x309**
## Description

Resolves: #948 

Ricochet is considered experimental, not easy to use and is no longer actively maintained.  See discussions #948  #840 

The latest commit on https://github.com/ricochet-im/ricochet was almost 2 years ago.
Also see:
https://github.com/ricochet-im/ricochet/issues/578
One of the lead developers is now focusing on another project:
https://openprivacy.ca/

That does not mean Ricochet is flawed, but it was always considered experimental and it is for advanced users only who need to setup and keep Tor up to date.

Wire was originally an IM recommendation, but then replaced with Riot.  Riot is not easy to make entirely private, but it is still a good option for advanced users.  So with one easy recommendation (Signal) which is primarily for mobile device use, that leaves currently two recommendations that are for advanced users (Riot and Ricochet). Since ease of use is supposed to be one of the primary considerations one of the advanced recommendations should be replaced with any easy to use option so that we have:
1 easy to use mobile only option (desktop client cannot be standalone, is tethered to a mobile)- Signal
1 easy to use mobile or desktop option- Wire
1 advanced option for mobile or desktop- Riot/Matrix.

Also Riot/Matrix has just been released out of beta and considered by the developers fully ready to use (though the encryption library may still be beta), so this puts Riot above the experimental status of Ricochet.

Up for discussion could be whether to mention Cwtch (still in alpha) as the probably replacement to Ricochet.

### CODE REVIEW

**Comment by ghbjklhv** [comment]
Possible Duplicate of #476

**Comment by ghbjklhv** [comment]
> Up for discussion could be whether to mention Cwtch (still in alpha) as the probably replacement to Ricochet.

What is Cwtch's URL? I cannot seem to find much information about it.

**Comment by Perelandra0x309** [comment]
> Wire has way to many flaws to be considered private.

Can you provide references for all the flaws you see in Wire?  We are aware of the meta data issue and already have a link for that on the website.  

> My personal opinion for sending confidential information is [Vuvuzela](https://vuvuzela.io/) (#637) which I believe would make a much better addition than Wire.

Please review https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md

>Software Criteria
>Easy to use. Could your mother use that tool or service? Usability is most important.
>Cross-platform / Accessible.
>Privacy respecting.
>Open Source / free software is preferred but not required.

Your suggestion while maybe "better" in some respects is not at all easy to use for average users and limited to those who understand software development concepts.  Please view possible suggestions for recommendation through the philosophy of this website and the target audience of average users.  Could your Mom and Dad use it?

**Comment by Perelandra0x309** [comment]
> What is Cwtch's URL? I cannot seem to find much information about it.

https://openprivacy.ca/work/cwtch/

**Comment by ghbjklhv** [comment]
@Perelandra0x309 It was just a general references to the issues with centralized services.
https://www.fsf.org/campaigns/priority-projects/decentralization-federation

Signal suffers from many non-free dependencies on mobile (see #922 )
I would assume Wire suffers from the same?
https://search.f-droid.org/?q=Wire&lang=en

File: _includes/sections/instant-messenger.html [hunk 1]
```diff
@@ -36,16 +36,20 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
...
-website="https://ricochet.im/"
-forum="https://forum.privacytools.io/t/discussion-ricochet/666"
-github="https://github.com/ricochet-im/ricochet"
+title="Wire"
+image="/assets/img/tools/wire.png"
+description='A free software End-to-End Encrypted chatting application that supports instant messaging, voice, and video calls. Full source code is available. <span class="badge badge-warning" data-toggle="tooltip" title="Wire stores metadata such as your contacts in plaintext (= not encrypted)."><a href="https://www.vice.com/en_us/article/gvzw5x/secure-messaging-app-wire-stores-everyone-youve-ever-contacted-in-plain-text">meta-data concerns</a> <i class="far fa-question-circle"></i></span>'
```
Comment (Mikaela) [inline _includes/sections/instant-messenger.html]: Why do you revert https://github.com/privacytoolsIO/privacytools.io/pull/950 and restore incorrect statement?

**Comment by Mikaela** [comment]
> It was just a general references to the issues with centralized services.

https://github.com/wireapp/wire-server/issues/631

> I would assume Wire suffers from the same?
https://search.f-droid.org/?q=Wire&lang=en

https://github.com/wireapp/wire-android/issues/5 has more details

**Comment by JonahAragon** [comment]
> What is Cwtch's URL? I cannot seem to find much information about it.

https://cwtch.im/

> Cwtch is an extension of the metadata resistant protocol Ricochet

This sounds to me like it's a new client (the successor to Ricochet) that's using the Ricochet *protocol* for communications. I'm not sure how ready it is or if it should be listed,

**Comment by blacklight447-ptio** [comment]
I've played around with Cwtch for a while, while it seems like a nice project, its definitely not ready for actual usage. I advise to keep an eye on it, but not add it till the project has matured more.

**Pull request merged**